src/esm_parser/esm_parser.py

```diff
@@ -71,11 +71,11 @@
 import subprocess
 import sys
 import warnings
-import numpy
```

Comment:
In conjunction with `black`, we could run `isort` which will automatically sort out imports.

Reply:
Yes, I am also implementing pytest.

Reply:
You don't need to implement `pytest`, it is already there. See here: 
https://github.com/esm-tools/esm_tools/blob/781c659c912c1bdefaecdff15e564b79c4a9e184/.github/workflows/esm_tools_actions.yml#L35-L36